vita3k/module/src/load_module.cpp

```diff
@@ -41,17 +41,19 @@ bool is_lle_module(SceSysmoduleModuleId module_id, EmuEnvState &emuenv) {
         SCE_SYSMODULE_HTTP,
         SCE_SYSMODULE_SSL,
         SCE_SYSMODULE_HTTPS,
+        SCE_SYSMODULE_SMART,
+        SCE_SYSMODULE_FACE
     };
 
     if (have_paths) {
         if (emuenv.cfg.current_config.modules_mode != ModulesMode::MANUAL) {
-            if (std::find(auto_lle_modules.begin(), auto_lle_modules.end(), module_id) != auto_lle_modules.end())
+            if (std::ranges::find(auto_lle_modules, module_id) != auto_lle_modules.end())
```

Comment:
good catch 👍

Reply:
std::ranges is still not compatible with the apple clang compiler